src/Function.cpp

```diff
@@ -562,6 +583,13 @@ Function::OutputForwardDecl(std::ostream &out)
 	OutputHeader(out);
 	if(func_attr_inline)
 		out << " __attribute__((always_inline))";
+	vector<Attribute*>::iterator itr;
+	for(itr = func_attr_generator.attributes.begin(); itr < func_attr_generator.attributes.end(); itr++)
```

Comment:
Call something like `func_attr_generator.GenerateAllAttributes()`/

Reply:
> Call something like `func_attr_generator.GenerateAllAttributes()`/

Got it.